content-api-post.js

```diff
@@ -55,14 +59,18 @@ exports.handler = function (event, context, callback) {
     .toBuffer()
     .then(processedImage => {
       const generatedName = `${uuidv4()}.${mimeType.ext}`;
-      const path = `${data.accountId}/${data.type}/${generatedName}`;
-      const metadata = {
-        SalesforceAccountId: data.accountId,
-      };
-      // All remaining metadata keys are optional. We can't append `null`s as this is not a valid value for headers.
+      const path = `${SalesforceId}/${data.type}/${generatedName}`;
+      const metadata = {};
+
+      // Assign the dynamic SF Id to the correct metadata
+      if (data.accountId) {
+        metadata.SalesforceAccountId = SalesforceId;
```

Comment:
I don't see any benefit in using the less descriptive, new variable here - we only need it for the S3 path setup. As it was before you could tell what the value is without having to grok another variable, and it does the same thing.